pandas/tests/frame/test_constructors.py

```diff
@@ -2102,18 +2114,22 @@ def test_constructor_categorical(self):
         )
         tm.assert_frame_equal(df, expected)
 
+    def test_from_nested_listlike_mixed_types(self):
         # mixed
         df = DataFrame([Categorical(list("abc")), list("def")])
         expected = DataFrame(
             {0: Series(list("abc"), dtype="category"), 1: list("def")}, columns=[0, 1]
         )
         tm.assert_frame_equal(df, expected)
 
+    def test_construct_from_listlikes_mismatched_lengths(self):
```

Comment:
may pay to split out this test file as getting kind of big

Reply:
ive been thinking that myself.  will address before long